spec/services/populi_service_spec.rb

```diff
@@ -130,11 +134,13 @@
         end
 
         it 'provides error message with wrong course_meeting_id' do
-          course_offering_id = "10547884"
-          enrollment_id = "76297621"
-          status = "PRESENT"
-          course_meeting_id = "531"
-          response = @populi.update_student_attendance(course_offering_id, enrollment_id, course_meeting_id, status)
+          current_academic_term = @populi.current_academic_term
+          updated_course_offering = @populi.courseofferings_by_term(current_academic_term[:id])[:data].first[:id]
+          enrollment_id = @populi.enrollments(updated_course_offering)[:data].first[:id]
+          status = "present"
```

Comment:
does all-caps/case matter here?

Reply:
Populi's API is not caps sensitive to the status message, so it would accept "PRESENT" or "present". We have initialized the variables in our testing in both ways to display the flexibility of the case matter.